tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -1688,4 +1691,37 @@ object TestingServer {
       treeViewProvider = Some(true),
       slowTaskProvider = Some(true)
     )
+
+  // In most cases lsp unit tests have the same classpath: one jvm + scala-library without any external deps
+  // Than means that in all these tests ClasspathSearch should be the same and we can calculate it once per run
+  // instead of once per test
+  val testingClasspathSearchIndexer: ClasspathSearch.Indexer =
```

Comment:
Added an additional improvement by reducing the time for ClasspathSearch indexing

Reply:
Aren't there any unit tests that have dependencies and the classpath search would be influenced by this?

Reply:
There are and they work fine with this change.
Cache is done by a key: dependency jars + excludedPackages conf + bucket size.
Unit test with a unique dependencies creates a new index but most of them reuses the default.

Reply:
Can we add your comment to the code comment?

> There are and they work fine with this change.
> Cache is done by a key: dependency jars + excludedPackages conf + bucket size.
> Unit test with a unique dependencies creates a new index but most of them reuses the default.

Reply:
updated this comment